### ISSUE TITLE
Accessibility analysis

### ISSUE THREAD

**Issue by bolonio** [primer/doctocat#502]
## Context

The doctocat theme needs an initial accessibility analysis, as well as Primer components accessibility use.

## Acceptance Criteria

- [ ] An accessibility analysis has been performed
- [ ] All accessibility issues have been created

**Comment by TheeMattOliver** [comment]
Hi @bolonio! 

FYI, if interested, I've proposed a PR on `react-simple-code-editor` (you will notice that live code components that are powered by `react-live` will fail WCAG success criteria 4.1.2 because of this).

This should at least prevent this component from violating the criterion and would improve accessibility in the Gatsby theme.

https://github.com/react-simple-code-editor/react-simple-code-editor/pull/104
